### ISSUE TITLE
must be use python2.7?

### ISSUE THREAD

**Issue by xuyue101012** [YadiraF/PRNet#104]
can I use python3.7?

**Comment by GewelsJI** [comment]
yes, I follow the author's envs, and I complement it successfully.

**Comment by GewelsJI** [comment]
yes, I follow the author's envs, and I complete it successfully.

**Comment by Ivy147** [comment]
I use py3.6,and it works.

**Comment by ljc19940403** [comment]
> yes, I follow the author's envs, and I complete it successfully.
hello，I may need your help
When I perform：python run_basics.py
2019-11-05 20:30:13.850630: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-11-05 20:30:13.989548: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:895] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-11-05 20:30:13.990105: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1105] Found device 0 with properties: 
name: Tesla P4 major: 6 minor: 1 memoryClockRate(GHz): 1.1135
pciBusID: 0000:00:07.0
totalMemory: 7.43GiB freeMemory: 7.32GiB
2019-11-05 20:30:13.990139: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1195] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: Tesla P4, pci bus id: 0000:00:07.0, compute capability: 6.1)
2019-11-05 20:30:17.550559: E tensorflow/stream_executor/cuda/cuda_dnn.cc:378] Loaded runtime CuDNN library: 7500 (compatibility version 7500) but source was compiled with 7004 (compatibility version 7000).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
2019-11-05 20:30:17.551302: F tensorflow/core/kernels/conv_ops.cc:717] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) Aborted
What is my problem with the unprocessed images in the folder？
